Automattic/wp-calypso
# CRO i2: plans grid new organization

#### Changes proposed in this Pull Request

* Include Jetpack Security Real-time and Jetpack Backup Real-time to the plans grid.
* Re-organize the cards in such a way that plans are always featured in the first row of the grid. 

#### Testing instructions

* Run this PR (Calypso Blue and Calypso Green builds).
* Visit the Plans page in Calypso Blue.
* Go to the abtest tool and select `v2 - slide outs` from `jetpackConversionRateOptimization` (see the animation below if you don't know how to do it).
* Make sure that regardless of the plan or products the site owns, the cards in the grid are always displayed in the following sequence:
```
1. Jetpack Security Daily
2. Jetpack Security Real Time
3. Jetpack Complete
4. Backup Daily
5. Back up real time
6. Anti spam
7. Scan
8. CRM
9. Search
10. Free
```
* Go to the abtest tool and select `v1 - 3 cols layout` from `jetpackConversionRateOptimization`.
* Make sure the grid looks exactly as it does in production.

* Repeat the exercise in Calypso Green (Pricing page).

Fixes 1196341175636977-as-1198561816278776

#### Demo
![Kapture 2020-10-22 at 12 24 53](https://user-images.githubusercontent.com/3418513/96894069-bcfccf00-1461-11eb-8742-a0c63ff42762.gif)

#### Calypso Blue
![image](https://user-images.githubusercontent.com/3418513/96894510-1402a400-1462-11eb-86e0-3c6ae57e1a0a.png)

#### Calypso Green
![image](https://user-images.githubusercontent.com/3418513/96894374-03eac480-1462-11eb-9ca6-a375d4048866.png)
